.changes/v3.9.0/1004-improvements.md

```diff
@@ -0,0 +1,4 @@
+* Resources `vcd_vapp_network` and `vcd_vapp_org_network` add convenience flag
+  `reboot_vapp_on_destroy`. When enabled, it will to power off parent vApp (and power back on after
```

Comment:
The more I read this, the more I think it should be called `reboot_vapp_on_delete` or `reboot_vapp_on_removal`, because "destroy" by Terraform definition implies deletion of **all** resources.
The `reboot_vapp_on_destroy` name implies that this argument is useful only for `terraform destroy` operation, but it is actually also useful when removing this single network instance from HCL.

Thoughts?

CC: @dataclouder @adambarreiro @adezxc

Reply:
I am open to renaming it to `reboot_vapp_on_delete`, but waiting for others to give perspective so that I don't have to change it twice (as we know it can be error prone :)

Reply:
While "destroy" doesn't necessarily imply destruction of all resources, as it can be run on targeted entities, it may give wrong ideas to the user. I am OK with both `reboot_vapp_on_removal` and `reboot_vapp_on_delete`.

Reply:
I agree with Adam, I like `reboot_vapp_on_removal` out of those as it's probably the best matching word for this action in my opinion.

Reply:
Will rename the field to `reboot_vapp_on_removal`

Reply:
Changed